deploy/olm-catalog/mobile-developer-console-operator/0.2.6/mobile-developer-console-operator.v0.2.6.clusterserviceversion.yaml

```diff
@@ -0,0 +1,236 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[{"apiVersion":"mdc.aerogear.org/v1alpha1","kind":"MobileDeveloperConsole","metadata":{"name":"example-mdc"},"spec":{"oAuthClientId":"mobile-developer-console","oAuthClientSecret":"foobar"}}]'
+    capabilities: Basic Install
+  name: mobile-developer-console-operator.v0.2.6
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Placeholder description
+      displayName: Mobile Developer Console Operator
+      kind: MobileDeveloperConsole
+      name: mobiledeveloperconsoles.mdc.aerogear.org
+      version: v1alpha1
+  displayName: Mobile Developer Console Operator
+  install:
+    spec:
+      deployments:
+      - name: mobile-developer-console-operator
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              name: mobile-developer-console-operator
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                name: mobile-developer-console-operator
+            spec:
+              containers:
+              - command:
+                - mobile-developer-console-operator
+                env:
+                - name: WATCH_NAMESPACE
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: POD_NAME
+                  valueFrom:
+                    fieldRef:
+                      fieldPath: metadata.name
+                - name: OPERATOR_NAME
+                  value: mobile-developer-console-operator
+                - name: OPENSHIFT_HOST
+                  value: ${OPENSHIFT_HOST}
```

Comment:
Where does this value come from when provisioned from the CSV through the OLM? When using the `deploy/operator.yaml`, one would need to have set this as an environment variable for the kubectl command I think.

Reply:
Yes, I set the `OPENSHIFT_HOST` env variable when the MDC CR is reconciled. https://github.com/jjaferson/integreatly-operator/blob/release/pkg/controller/installation/products/mobiledeveloperconsole/reconciler.go#L139

Reply:
Perfect, thanks. :+1: